Bootstraps/Symfony.php

```diff
@@ -137,6 +137,7 @@ public function postHandle($app)
             if (!$em->getManager()->isOpen()) {
                 $em->resetManager();
             }
```

Comment:
Else clear sinc reset would surely clear it, too?

Reply:
Good point. Its changed